Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,243 @@
+                            {
+
+                                highestBalanceHolder += ($" and {currentBalanceHolder}");
+
+                            }
+                            else
+                            {
+                                highestBalanceHolder += ($", {currentBalanceHolder}");
+                                holderCount++;
+                            }
+                        }
+                    }
+                }
+            }
+            string stringHighestBalance = highestBalance.ToString();
+            return ($"{highestBalanceHolder} had the most money ever. ¤{stringHighestBalance}.");
         }
 
         /// <summary>
         /// Return name and loss of a person with a biggest loss (balance change negative).
         /// </summary>
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            float biggestLoss = 0f;
+            float balance;
+            float nextBalance;
+            string biggestLoser = "";
+            int holderCount = 0;
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+            foreach (string entry in peopleAndBalances)
+            {
+                string[] userHistory = entry.Split(',');
+                if (userHistory.Length <= 2)
+                {
+                    return "N/A.";
+                }
+                string currentBalanceHolder = userHistory[0];
+
+                for (int i = 1; i < userHistory.Length - 1; i++)
+                {
+                  
+                    balance = float.TryParse(userHistory[i], out float result) ? float.Parse(userHistory[i]) : float.NaN;
+
+                    nextBalance = float.TryParse(userHistory[i + 1], out float nextResult) ? float.Parse(userHistory[i + 1]) : float.NaN;
+
+                    biggestLoss = ((nextBalance - balance) < biggestLoss) ? (nextBalance - balance) : biggestLoss;
+
+
+
+                    if (biggestLoss == nextBalance - balance && biggestLoser != currentBalanceHolder)
+                    {
+                        switch (holderCount)
+                        {
+                            case 0:
+                                {
+                                    biggestLoser = currentBalanceHolder;
+                                    holderCount++;
+                                    break;
+                                }
+
+                            case 1:
+                                {
+                                    biggestLoser += ($", {currentBalanceHolder}");
+                                    holderCount++;
+                                    break;
+                                }
+                            case 3:
+                                {
+                                    biggestLoser += ($" and {currentBalanceHolder}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4953253)